src/cart/crt_iv.c

```diff
@@ -1061,7 +1087,10 @@ handle_ivfetch_response(const struct crt_cb_info *cb_info)
 	struct ivf_key_in_progress	*kip_entry;
 	uint32_t			class_id;
 	int				rc;
+	crt_iv_ver_t			rpc_version = input->ifi_grp_ver;
+	crt_iv_ver_t			local_version;
```

Comment:
group version (local_version) in this case is not of crt_iv_ver_t type. it is same type as in crt_grp_priv::gp_version

Reply:
Changed local_version type to uint32_t.